Obsidian/Commands/MainCommandModule.cs

```diff
@@ -272,14 +280,13 @@ public async Task TeleportAsync(CommandContext Context, [Remaining] VectorF loca
         [Command("op")]
         [CommandInfo("Give operator rights to a specific player.", "/op <player>")]
         [RequirePermission]
+        [IssuerScope(CommandIssuers.Any)]
         public async Task GiveOpAsync(CommandContext Context, IPlayer player)
         {
-            var server = (Server)Context.Server;
-            var onlinePlayers = server.OnlinePlayers;
-            if (!onlinePlayers.ContainsKey(player.Uuid) || !onlinePlayers.Any(x => x.Value.Username == player.Username))
+            if (player == null)
```

Comment:
isn't player also null when issuer is the console? why have CommandIssuers.Any when consoles probably can't even execute it?

Reply:
🤔  `player` it's is the player to whom the op, not an command issuer